src/pages/project/project.jsx

```diff
@@ -82,7 +82,11 @@ var Project = React.createClass({
     // FIXME: TODO: This should be done by preventDefaulting the touch event, not via CSS.
     document.body.style.overflowY = 'hidden';
     var mode = this.state.params.mode;
-    var isPlayOnly = (mode === 'play' || mode === 'link');
+
+    var isPlayOnly = mode === 'play';
+    var isLink = mode === 'link';
+    var isPlayOrLink = isPlayOnly || isLink;
```

Comment:
Looking at the getRemovePageButton function, as well as what we use this var for on line 101 in this file, we're using it exclusively as `isPlayOrLink || !this.state.selectedEl`, so it might make sense to pull that up as a real var:
```
var disableMainMenu = isPlayOrLink || !this.state.selectedEl;
...
{ this.getRemovePageButton(disableMainMenu) }
...
<PrimaryButton url={ this.getPageURL(this.state.params, this.state.selectedEl) } off={disableMainMenu} ...
...
```